### DISCUSSION TITLE
Nextjs inbuilt Images are overlapping?

### DISCUSSION THREAD

**Discussion by 18121A05L2** [tailwindlabs/tailwindcss#9189] [Help]
hey can anyone help me with this  ?

https://stackoverflow.com/questions/73499353/nextjs-inbuilt-images-are-overlapping

**Comment by wongjn** [comment]
> […] after using these, only the Nextjs `<Image>` is overlapping on top of my header , I had 2-3 images […]

When you say "only the Nextjs `<Image>` is overlapping", do you mean that the native `<img>` image elements are not `fixed` positioned? Or could they be underneath the `<Image>` element, which is why you tried `-z-1`? As for `-z-1`, this is not a valid Tailwind class with default Tailwind configuration. As per [the documentation](https://tailwindcss.com/docs/z-index), you could try `-z-10` instead.

**Reply by 18121A05L2** [reply]
I tried with `-z-10` too , but for visualization one image is behaving normal and other one is invisible and that behaving as normal  had one more issue

--> i actually had a blog post and nextjs `<Image>` is present in that and i used Link for the blog post , and in the header I had signin button , if scroll and position both at same place and click on the signin button , the post below where the `<Image>` is positioned is triggering and it made routing to that link

**Reply by wongjn** [reply]
I'm sorry but I can't quite understand what you have written, could you rephrase? Perhaps an example could help explain your problem too?

**Reply by 18121A05L2** [reply]
here is the git link --> https://github.com/18121A05L2/Medium-Nextjs-clone
deployed link --> https://medium-nextjs-clone.vercel.app/

**Reply by 18121A05L2** [reply]
its working bro , i used z-100 for header , thats the wrong one
